software/paie52.py

```diff
@@ -430,7 +430,8 @@ def setup(self):
                     'zlib-devel boost-system libgfortran boost-python boost-thread '
                     'boost-filesystem java-1.8.0-openjdk-devel scipy PyYAML '
                     'pyparsing python-pillow python-matplotlib pciutils libgcc '
-                    'libgomp libstdc++ libstdc++-devel cpp gcc-c++')
+                    'libgomp libstdc++ libstdc++-devel cpp gcc-c++ '
+                    'dejavu-serif-fonts')
```

Comment:
I tested a33e2dc and 'dejavu-serif-fonts' seemed to install fine (the install task passed, but I did _not_ verify anything was actually installed) so I'm not sure it's needed here.